indico/modules/events/sessions/operations.py

```diff
@@ -19,15 +19,19 @@
 from flask import session
 
 from indico.core.db import db
+from indico.modules.events.logs.models.entries import EventLogRealm, EventLogKind
 from indico.modules.events.sessions import logger
 from indico.modules.events.sessions.models.sessions import Session
-from indico.modules.events.logs.models.entries import EventLogRealm, EventLogKind
 
 
 def create_session(event, data):
     """Create a new session with the information passed in the `data` argument"""
     event_session = Session(event_new=event)
+    session_contrib_id = data.pop('session_contrib')
```

Comment:
Pretty ugly IMO. I'd rather send a second ajax request after the session has been created (just like you'd to when selecting a session manually)